src/EnergyPlus/HeatBalanceKivaManager.cc

```diff
@@ -695,8 +695,8 @@ bool KivaManager::setupKivaInstances()
 					exposedFraction = expPerimMap[surfNum].exposedFraction;
 				}
 			} else {
-				userSetExposedPerimeter = false;
-				useDetailedExposedPerimeter = true;
```

Comment:
Tested ZoneCoupledKivaWalkoutBasement with the new SurfaceProperty:ExposedFoundationPerimeter object deleted, just to see the error message work, but it crashed on line 707 because `userSetExposedPerimeter` was uninitialized.  Probably best to initialize all of the bools to false when declared up in lines 680:683.